# DNA-methylation-analysis


### papers
*[MINI REVIEW: Statistical methods for detecting differentially methylated loci and regions](http://biorxiv.org/content/biorxiv/early/2014/07/15/007120.full.pdf)

**Illumina is phasing out 450k and introducing the new 850k as in the end of 2015**
### 450k array analysis
* [bioconductor missMethyl](https://www.bioconductor.org/packages/release/bioc/html/missMethyl.html)  
* [450k-analysis-guide](https://github.com/crazyhottommy/450k-analysis-guide) A bit dated, but still useful.
* [bioconductor MEAL](http://bioconductor.org/packages/devel/bioc/vignettes/MEAL/inst/doc/MEAL.html) MEAL aims to facilitate the analysis of Illumina Methylation 450K chips   
* [an R package for normalization of DNA methylation data when there are multiple cell or tissue types.](https://github.com/GreenwoodLab/funtooNorm)
* [normalize450K bioconductor](http://bioconductor.org/packages/devel/bioc/html/normalize450K.html)

### RRBS
* [M3D: Statistical Testing for RRBS Data Sets](https://www.bioconductor.org/packages/release/bioc/vignettes/M3D/inst/doc/M3D_vignette.pdf)  

* [methylkit](https://github.com/al2na/methylKit) deals with RRBS  data.

* [BiSeq](https://www.bioconductor.org/packages/release/bioc/html/BiSeq.html) RRBS centric.

### Whole genome BS-seq
* [MethGo: a comprehensive tool for analyzing whole-genome bisulfite sequencing data](http://paoyangchen-laboratory.github.io/methgo/) 
* [bismark-pipeline.pdf](https://github.com/crazyhottommy/DNA-methylation-analysis/files/92245/bismark-pipeline.pdf)
* [Post-Processing Bismark Bisulphite Sequencing Data](http://blog.mcbryan.co.uk/2013/02/post-processing-bismark-bisulphite.html)  
* [The Binomial Distribution, Python and Bisulphite Sequencing](http://blog.mcbryan.co.uk/2013/02/the-binomial-distribution-python-and.html) 

### Clustering
* [Clustering of (epigenetic) DNA methylation data using a variational Bayes NMF algorithm:EpiCluster bioconductor ](http://bioconductor.org/packages/devel/bioc/html/EpiCluster.html)
